data/h2o2.yaml

```
@@ -37,6 +46,10 @@ species:
     diameter: 2.92
     polarizability: 0.79
     rotational-relaxation: 280.0
+  equation-of-state:
+    model: Redlich-Kwong
+    a: [1.43319e+11, 0]
+    b: 18.42802577
```

I know these are written this way in `nDodecane_Reitz.yaml`, but if the second term in the `a` coefficient is zero, you can just write this as:
```suggestion
    a: 1.43319e+11
    b: 18.42802577
```
I think this would be better, since the two-term model for "a" is kind of an oddity in Cantera's implementation of R-K.

---

Updated.